vscode-extension/src/features/treeView/assignmentExplorer/nodes/CollisionNode.ts

```diff
@@ -18,13 +19,14 @@ export class CollisionNode extends AppAwareDescendantNode implements AssignmentI
     protected _includeLogicalNameInDescription = false;
     protected _includeLogicalNameInLabel = false;
 
-    constructor(parent: AppAwareNode, object: ALObject) {
+    constructor(parent: AppsAwareNode, object: ALObject) {
         super(parent);
         this._object = object;
         this._uri = Uri.file(object.path);
         this._label = object.path.split(/[\\|\/]/).pop()!;
         this._description = object.id.toString();
-        this._contextValues.push(ContextValues.StoreAssignment);
+        if (this.apps.every(app => app.config.appPoolId === this.apps[0].config.appPoolId))
```

Comment:
Curly braces. Always with `if`

Reply:
Done in 7ba87ccc7cdb1b45172d51296ded88b2e5128580